docs/semgrep-cloud-platform/dashboard.md

```diff
@@ -65,14 +65,66 @@ To change the projects filter:
 1. Click the <i class="fa-solid fa-sliders"></i> **All projects** button.
 2. Select the project(s) from the drop-down box. The Dashboard, including all widgets, reloads to reflect data from the selected project(s).
 
-## Summarizing a project’s security posture
+## Summarizing a projects security posture
```

Comment:
What's the reason for this edit? I don't think it makes any sense?

Reply:
```suggestion
## Summarizing a project security posture
```

Reply:
Sure go ahead and commit!